composer.json

```diff
@@ -54,10 +54,10 @@
     "require-dev": {
         "aws/aws-sdk-php": "2.7.1",
         "phpunit/phpunit": "~4.8",
-        "facebook/xhprof": "dev-master",
         "phpseclib/phpseclib": "~0.3.8",
         "symfony/var-dumper": "~2.6.0",
-        "symfony/yaml": "~2.6.0"
+        "symfony/yaml": "~2.6.0",
+        "lox/xhprof": "dev-master"
```

Comment:
could you also remove the package definition for `facebook/xhprof` the lines below (63-88).

Reply:
I've removed them.